src-java/northbound-service/northbound/src/test/java/org/openkilda/northbound/converter/LinkMapperTest.java

```diff
@@ -76,7 +76,8 @@ public void testMapResponse() {
                 false,
                 true,
                 BFD_SESSION_STATUS,
-                PACKET_ID);
+                PACKET_ID,
+                null);
```

Comment:
Why not to test the description mapping?

Reply:
+1

Reply:
Updated.